### PR TITLE
Docs update jinja

### PR DESCRIPTION
**Pull request by tthebst**
Hi, I update the Jinja2 part in the views documentation(Issue #87). I added an line statement example to each Jinja2 example for clarification. I also tried to add some documentation.

### CODE REVIEW

**Review by josephmancuso — APPROVED** [review]

File: the-basics/views.md [hunk 1]
```diff
@@ -504,28 +504,29 @@ Get the URL to a location:
 
 ## Jinja2
 
-Below is example Jinja2 syntax which Masonite uses to build views
+Below are some examples of the Jinja2 syntax which Masonite uses to build views. 
 
 ### Line Statements
 
-It's important to note that all syntax can be rewritten with line statements.
+It's important to note that Jinja2 statements can be rewritten with line statements and line statements are preferred in Masonite. In comparison to Jinja2 line statements evaluete the whole line, thus the name line statement.
```
Comment (assertchris) [inline the-basics/views.md:511]: evaluete

**Review by assertchris — COMMENTED** [review]
While I think this documentation is much better than what was previously there; I think one of the main things I had hoped would happen is for a recommended approach to be clear. That is, that we'd promote line statements except in the edge case of having a single-line flow control statement (which would require the Jinja syntax). As this shows both, exhaustively; it leaves the reader wondering which is "the right one" to use.

**Comment by assertchris** [comment]
I'm sorry I am so late to the party. I was on holiday until quite recently, and catching up has proven to be challenging.

**Comment by tthebst** [comment]
@assertchris hi. How do you think i should highlight that the line statement syntax is preferred. Should i add something like a preferred tag (Line statements(preferred)). One idea is to show the line statement expression first and write sth like "alternatively in Jinja2"

**Pull request merged**